src/app/forms/just-write-it/just-write-it.jade

```diff
@@ -0,0 +1,19 @@
+.form
+  .background--image
+    img(src="../../assets/images/home_background.png")
+  .outer
+    .middle.text-center
+      .inner
+        form(novalidate=true, name="uvpForm", ng-hide="ctrl.showUVP")
+          .row
+            .small-12.columns
+              textarea(placeholder="Just write it!", ng-model="ctrl.method.prose", required=true)
+          .row
+            span.has-tip(data-tooltip, aria-haspopup=true, title= "Avoid rummors and unclear information with a better publisher network") Example
+            br
+            button.button(ui-sref="ZagFormat") Previous
+            button.button(ng-disabled="!uvpForm.$valid", ng-click="ctrl.display()") Go
+        .row(ng-show="ctrl.showUVP")
+          h1 {{ctrl.method.template()}}
+          button.button(ng-click="ctrl.back()") Back
+.footer #14 Just write
```

Comment:
This is how the template looks like
![image](https://cloud.githubusercontent.com/assets/8473858/7381511/3af05a26-edc1-11e4-9d59-09a7a861f469.png)